plugins/priceFeed.go

```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/interstellar/kelp/model"
 )
 
+// PrivateSdexHack is a temporary hack variable for SDEX price feeds pending refactor
+var PrivateSdexHack *SDEX
```

Comment:
with the comment below on sdex.go#GetOrderBook, we should be constructing the SDEX instance directly in the case statement below. i.e. we no longer need the global variable for SDEX but will instead need a global variable for the dependencies needed to instantiate a new SDEX instance:
- api *horizon.Client
- network build.Network

you can keep all this in one global variable struct if you want, calling it PrivateSdexHack like is already done.

    var PrivateSdexHack struct {
        api *horizon.Client
        network build.Network
    }

This will ensure that your logic in newSDEXFeed calls the `api` and `network` values on PrivateSdexHack (it probably throws a compile error now?)

Reply:
Nope, it compiles and runs. I do have it explicitly grabbing  `.network` and `.api` so that may be why it works.

Reply:
Change made